### PR TITLE
chore: ran code gen script for playground:documents

### PR DESCRIPTION
**Pull request by arnels**
Ran the code gen script for playground:documents to include the latest API definitions.

Jira task: https://cognitedata.atlassian.net/browse/SBS-4803

### CODE REVIEW

File: packages/playground/src/api/documents/types.gen.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Do not modify this file!
```
Comment (henrist) [inline packages/playground/src/api/documents/types.gen.ts:1]: Can you change this to be same as https://github.com/cognitedata/cognite-sdk-js/blob/4394e60273fa4fe33be731289beb2b09dac57f31/packages/stable/src/api/documents/types.gen.ts#L1-L4 ?

**Review by henrist — CHANGES_REQUESTED** [review]

File: packages/playground/src/api/documents/types.gen.ts (hunk 1 above)
Comment (arnels) [reply packages/playground/src/api/documents/types.gen.ts:1]: Fixed

**Review by henrist — APPROVED** [review]

**Comment by henrist** [comment]
Flaky tests - I triggered a rerun

**Comment by andersfylling** [comment]
I wonder if the context team did a breaking change in their api. The error complains about a type issue in the request.

**Comment by andersfylling** [comment]
@cognitedata/contextualization could you take a look?

**Comment by henrist** [comment]
> I wonder if the context team did a breaking change in their api. The error complains about a type issue in the request.

Ah, good spot! I just assumed flakyness a bit too quick.

Maybe the test itself is originally wrong/invalid?

**Comment by andersfylling** [comment]
> 

Yeah, integration tests in this SDK are testing a bit too much internal behavior rather than just if the endpoint + types are correct (+ timeout issues). So they often get flaky.

These used to work before, I believe @cyriaquebrousse implemented them.

**Comment by henrist** [comment]
I didn't pay attention, but next time use "fix" for this change and it will also auto-release. :+1:

**Pull request merged**